domains/kiwi.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "imdakiki",
+        "email": "skullkid73@outlook.com"
+    },
+    "record": {
+        "CNAME": "kikisdevelopment.github.io/kikisite/"
```

Comment:
it can't contain `/`, you should remove `/kikisite/`

Reply:
Done

Reply:
What do I do now?

Reply:
I updated the JSON